src/parameter_search.ipynb

```diff
@@ -320,11 +320,11 @@
    "outputs": [],
    "source": [
     "searchable_index_params = {\n",
-    "    'chunk_size' : [192, 256, 320],\n",
-    "    'chunk_overlap' : [64, 128, 192],\n",
+    "    'chunk_size' : [128, 192, 256, 320],\n",
+    "    'chunk_overlap' : [32, 64, 96, 128, 160, 192],\n",
     "}\n",
     "searchable_shared_params = {\n",
-    "    'embed_model' : [(\"huggingface\",\"all-MiniLM-L6-v2\")]\n",
+    "    'embed_model' : [(\"huggingface\",\"all-MiniLM-L6-v2\"),(\"huggingface\", \"BAAI/bge-base-en-v1.5\")]\n",
```

Comment:
why are we re-introducing the `bge` model again?

Reply:
will add it as a comment